src/templates/Footer/index.js

```diff
@@ -236,6 +239,12 @@ Footer.propTypes = {
    */
   className: PropTypes.string,
 
+  /**
+   * This prop will conditionally render the live chat button on mobile
+   */
```

Comment:
Should there be whitespace between the comment and the prop? Not super familiar with the syntax - but the other comments don't have that

Reply:
oh yeah i can update this!